.simplecov

```diff
@@ -0,0 +1,14 @@
+SimpleCov.start 'test_frameworks' do
+  enable_coverage :branch
+  add_group "Aggregations", "lib/arelastic/aggregations"
+  add_group "Arities", "lib/arelastic/arities"
+  add_group "Builders", "lib/arelastic/builders"
+  add_group "Mappings", "lib/arelastic/mappings"
+  add_group "Nodes", "lib/arelastic/nodes"
+  add_group "Queries", "lib/arelastic/queries"
+  add_group "Searches", "lib/arelastic/searches"
+  add_group "Sorts", "lib/arelastic/sorts"
+end
+
+SimpleCov.minimum_coverage line: 97.8, branch: 97.2
+SimpleCov.minimum_coverage_by_file line: 70, branch: 92.8
```

Comment:
Do these rules break the build if violated, or just change the report format?

Reply:
They break it.  My intent is to ensure that contributors can't commit new code without writing tests for it.